src/main.js

```diff
@@ -81,9 +95,11 @@ function displayLoggedActivity() {
       <p class="activity-description">${savedActivities[i].description}</p>
     </section>
     <section class="category-color-box">
-      <div class="category-color-bar ${savedActivities[i].category}"></div>
+      <div class="category-color-bar"></div>
```

Comment:
It's great that y'all are refactoring and thinking of different ways to do this. I wonder if `currentActivity` or `savedActivities[i]` was part of the className for these bar colors, there would be a solution here that used that interpolation. 

Instead of `red-bar`, would `meditate-bar` make a difference?